packages/devextreme/testing/tests/DevExpress.ui.widgets/tabs.tests.js

```diff
@@ -318,6 +326,19 @@ QUnit.module('General', () => {
             assert.ok($element.hasClass(TABS_ICON_POSITION_CLASS[iconPosition]));
         });
     });
+
+    QUnit.test('the tabs element must have a correct styling mode class', function(assert) {
+        const $element = $('#tabs').dxTabs({ items: [1, 2, 3] });
+        const instance = $element.dxTabs('instance');
+
+        assert.strictEqual(instance.option('stylingMode'), STYLING_MODE.primary);
+        assert.strictEqual($element.hasClass(TABS_STYLING_MODE_CLASS.primary), true);
```

Comment:
We don't check that other styling mode classes not added to element.

Reply:
Fixed here [48547431cb91e50f04bb940df4b4dd7095916eaa]